plasmapy/atomic/particle_class.py

```diff
@@ -608,6 +609,35 @@ def ionic_symbol(self) -> Optional[str]:
         """
         return self._attributes['ion']
 
+    @property
+    def roman_symbol(self) -> Optional[str]:
+        """
+        Return the spectral name of the particle (i.e. the ionic symbol in
```

Comment:
I think the docstring can be expanded a bit. If you look in other parts of the code we explicitly include argument names, acceptable types, a short description of the argument, and likewise for the returned value(s).

Reply:
This docstring is pretty similar in length and in detail to many of the other docstrings for attributes in the `Particle` class, since as an attribute it doesn't take in any arguments.  The only thing I can think of would be to add that the Roman numeral represents one plus the integer charge, and that it'll raise a `ChargeError` if no charge has been specified and a `roman.OutOfRangeError` if the charge is negative.

Reply:
Thanks for clarifying about the lack of arguments, I totally missed the fact that it was an attribute.

Do we currently have an error check for if the user inputs a charge which is beyond the atomic number for the atom/isotope? If not, it should be included under `ChargeError`.

Reply:
Yep, if we do something like `Particle('H++')` it will yield an `InvalidParticleError`.  `Particle` also accepts negatively charged ions, but issues an `AtomicWarning` for integer charges less than -3 or so.  Maybe we should do an `InvalidParticleError` for impossibly negatively charged ions as well.

Reply:
Perhaps we should include an `InvalidLanguageError` for `Particle("C++")`.